src/cljs/rems/application.cljs

```diff
@@ -615,7 +616,7 @@
                        (when-let [mail (:email attributes)]
                          [info-field (text :t.applicant-info/email) mail {:inline? true}])
                        (when-let [organizations (:organizations attributes)]
-                         [info-field (text :t.applicant-info/organization) (str/join ", " (map :organization/name organizations)) {:inline? true}])]
+                         [info-field (text :t.applicant-info/organization) (str/join ", " (map (comp language :organization/name) organizations)) {:inline? true}])]
```

Comment:
check this in a browser test please

Reply:
Done, and found some relevant problems to fix while at it.